features/gba-dev/overlijden-gba.feature

```diff
@@ -0,0 +1,127 @@
+      En de persoon heeft de volgende 'overlijden' gegevens
+      | datum overlijden (08.10) | plaats overlijden (08.20) | land overlijden (08.30) |
+      | 20020701                 | 0518                      | 6030                    |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                       | waarde                              |
+      | type                       | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam              | Jansen                              |
+      | geboortedatum              | 1963-04-05                          |
+      | inclusiefOverledenPersonen | true                                |
+      | fields                     | burgerservicenummer                 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 000000139 |
+      | overlijden.datum              | 19630405  |
+
+    Scenario: overleden persoon wordt geraadpleegd met burgerservicenummer en datum overlijden veld wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende 'overlijden' gegevens
+      | datum overlijden (08.10) | plaats overlijden (08.20) | land overlijden (08.30) |
+      | 20020701                 | 0518                      | 6030                    |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                               |
+      | type                | RaadpleegMetBurgerservicenummer      |
+      | burgerservicenummer | 000000139                            |
+      | fields              | burgerservicenummer,overlijden.datum |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000139 |
+      En heeft de persoon de volgende 'overlijden' gegevens
+      | naam               | waarde      |
+      | datum              | 20020701    |
+
+
+    Abstract Scenario: overleden persoon wordt geraadpleegd met burgerservicenummer, <field> overlijden is onbekend en wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende 'overlijden' gegevens
+      | naam                     | waarde   |
+      | datum overlijden (08.10) | 20020701 |
+      | <naam gba element>       | 0000     |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                 |
+      | type                | RaadpleegMetBurgerservicenummer        |
+      | burgerservicenummer | 000000139                              |
+      | fields              | burgerservicenummer,overlijden.<field> |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                            | waarde    |
+      | burgerservicenummer             | 000000139 |
+      | overlijden.datum                | 20020701  |
```

Comment:
in fields wordt niet gevraagd niet naar overlijden.datum, dan zou het niet in de response mogen voorkomen.

Reply:
Als ik nu regel 5 en 6, dan lijkt het wel weer te kloppen dat overlijden.datum in de response moet voorkomen. Maar ik wil het even checken, want de proxy kan indicatieOverleden ook op true zetten wanneer één van de overlijden velden wordt geretourneerd